src/app/features/vendor/vendor-creation/vendor-creation.component.html

```diff
@@ -116,10 +192,23 @@ <h2>New vendor</h2>
                     placeholder="Enter Number"
                     [class.invalidInput]="
                       addVendorForm.get('phone')?.invalid &&
-                      addVendorForm.get('phone')?.touched
-                    "
+                      addVendorForm.get('phone')?.touched"
                   />
                 </p-inputGroup>
+                <div
```

Comment:
use a common template for showing error messages

Reply:
done